Source/carthage/CopyFrameworks.swift

```diff
@@ -93,3 +129,7 @@ private func inputFiles() -> SignalProducer<String, CarthageError> {
 				|> flatten(.Concat)
 		}
 }
+
+private func buildAction() -> String? {
```

Comment:
How about naming `isArchiveBuildAction` or something and returning `Bool` as with `codeSigningAllowed()`?